# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

Tag_poseActionGoal action_goal
Tag_poseActionResult action_result
Tag_poseActionFeedback action_feedback
